cli/exec.go

```diff
@@ -54,6 +55,7 @@ func (e *ExecCommand) addFlags() {
 	flagSet.BoolVarP(&e.Interactive, "interactive", "i", false, "Open container's STDIN")
 	flagSet.StringVarP(&e.User, "user", "u", "", "Username or UID (format: <name|uid>[:<group|gid>])")
 	flagSet.StringArrayVarP(&e.Envs, "env", "e", []string{}, "Set environment variables")
+	flagSet.BoolVar(&e.Privileged, "privileged", false, "Give extended privileges to the exec process")
```

Comment:
Do we give `extended`  or `all` capabilities to the exec process ?

Reply:
We give all capabilities, same with `run container`, the code in daemon show this